great_expectations/datasource/fluent/sqlite_datasource.py

```diff
@@ -215,9 +215,3 @@ def add_query_asset(
         )
 
     add_query_asset.__doc__ = SQLDatasource.add_query_asset.__doc__
-
-
-# Removed automatically added add_*_asset methods we don't want.
-# TODO: Prevent these from being created.
-delattr(SqliteDatasource, "add_sqlite_table_asset")
```

Comment:
Oh nice, we can get rid of these now. I wonder if it was this that was causing the issue?